RecoCTPPS/TotemRPLocal/plugins/CTPPSDiamondLocalTrackFitter.cc

```diff
@@ -102,11 +102,41 @@ CTPPSDiamondLocalTrackFitter::produce( edm::Event& iEvent, const edm::EventSetup
 void
 CTPPSDiamondLocalTrackFitter::fillDescriptions( edm::ConfigurationDescriptions& descr )
 {
-  // The following says we do not know what parameters are allowed so do no validation
-  // Please change this to state exactly what you do use, even if it is no parameters
   edm::ParameterSetDescription desc;
-  desc.setUnknown();
-  descr.addDefault( desc );
+  desc.add<edm::InputTag>( "recHitsTag", edm::InputTag( "ctppsDiamondRecHits" ) );
+  desc.add<int>( "verbosity", 0 );
+
+  edm::ParameterSetDescription trackingAlgoParams;
+  trackingAlgoParams.add<double>( "threshold", 1.5 );
+  trackingAlgoParams.add<double>( "thresholdFromMaximum", 0.5 );
+  trackingAlgoParams.add<double>( "resolution", 0.01 /* mm */ );
+  trackingAlgoParams.add<double>( "sigma", 0.1 );
+  trackingAlgoParams.add<double>( "startFromX", -0.5 /* mm */ );
+  trackingAlgoParams.add<double>( "stopAtX", 19.5 /* mm */ );
+
+  trackingAlgoParams.add<std::string>( "pixelEfficiencyFunction", "(TMath::Erf((x-[0]+0.5*[1])/([2]/4)+2)+1)*TMath::Erfc((x-[0]-0.5*[1])/([2]/4)-2)/4" );
```

Comment:
Hi @perrotta 
We would prefer to maintain the current high-flexibility definition of this efficiency function for now, as the diamond detector is still under heavy commissioning. Of course, as soon as this phase will be over, we will indeed switch to a more conventional way of defining these functions (i.e. even hardcoding this formula at the plugin level).